README.md

```diff
@@ -188,6 +188,25 @@ sudo xbps-install -S SDL2-devel SDL2_image-devel SDL2_net-devel alsa-lib-devel \
                      speexdsp-devel libatomic-devel libXi-devel
 ```
 
+``` shell
+# Nix 
+# With Home Manager on home.nix (Recommended Permanent Installation)
+home.packages = [ pkg-config gcc_multi cmake ccache SDL2 SDL2_image SDL2_net \ 
+                  fluidsynth glib gtest libGL libGLU libjack2 libmt32emu libogg \ 
+                  libpng libpulseaudio libslirp libsndfile meson ninja opusfile \
+                  libselinux speexdsp stdenv alsa-lib xorg.libXi irr1 ]
+
+# With the standard NixOS Configuration on configuration.nix
+environment.systemPackages = [ 
+                  pkg-config gcc_multi cmake ccache SDL2 SDL2_image SDL2_net \ 
+                  fluidsynth glib gtest libGL libGLU libjack2 libmt32emu libogg \ 
+                  libpng libpulseaudio libslirp libsndfile meson ninja opusfile \
+                  libselinux speexdsp stdenv alsa-lib xorg.libXi irr1 
+                  ]; 
+                  
+# See the build-nix.md for more information
```

Comment:
Should link to build-nix.md be added here?

Reply:
We don't have links for the other projects, so this would be an inconsistent change (if we do this, then we should add links for all the other one-off build notes).

I think the link from the `BUILD.md` is fine.. for users interested, they will definitely be reading the BUILD.md guild anyway if they're down this path.

Reply:
Ok!